scipy/stats/_binned_statistic.py

```diff
@@ -280,6 +280,10 @@ def binned_statistic_dd(sample, values, statistic='mean',
             referenced.
           * 'sum' : compute the sum of values for points within each bin.
             This is identical to a weighted histogram.
+          * 'max' : compute the maximum value for points within each bin.
+            Empty bins will be represented by NaN.
+          * 'min' : compute the minimum value for points within each bin.
+            Empty bins will be represented by NaN.
```

Comment:
These new options should also be listed in the docstrings of `binned_statistic_2d` and `binned_statistic`.